index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Abraham Portfolio</title>
+</head>
+
+<body>
+  <main class="container">
+    <!-- header section -->
+    <header class="nav-container">
+      <a href="#" id="nameLogo">Cabraham</a>
+      <a href="#" class="imgLogo"><img class="imgLogo" src="./assets/icons/Union.svg" alt="line-menu"></a>
+    </header>
+
+    <!-- intro section  -->
+
+    <div class="intro">
+      <!-- main text content -->
+      <div class="text-container">
+        <h1 class="primary-text">I’m Abraham <br>Glad to see <br>you!</h1>
+        <!-- dividerText -->
+        <div class="dividerText"></div>
+        <p class="secondary-text">I’m a software developer! I can help you build a product , feature or website Look
+          through some of my work and <br>experience! If you like what you see and have a project you need coded, don’t
+          hestiate
+          to contact me.</p>
+      </div>
+      <!-- dividerText -->
+      <div class="dividerText"></div>
+
+      <!-- call to action text -->
+      <div class="link-container">
+        <h1 class="link-text">LET’S CONNECT</h1>
+      </div>
```

Comment:
- [ ] The 'let's connect' heading should be wrapped with a different heading(`hx`) tag. The main title of the page should be wrapped with the `h1` tag and this is not the main title of the page. Kindly change that to an `h3` tag fo better semantics.

Reply:
thanks done